src/boot/efi/boot.c

```diff
@@ -112,14 +112,19 @@ typedef struct {
  * The other values may be set by systemd-boot itself and changing those
  * will lead to functional regression when new version of systemd-boot is
  * installed.
+ *
+ * All the 64bit values are not ABI and will never be written to an efi
+ * variable.
```

Comment:
interesting idea. maybe using signed 64bit might be even nicer? but doesn't matter, this works too